style.css

```diff
@@ -91,3 +91,225 @@ nav ul li a {
 .links ul li a {
   color: white;
 }
+
+.works-Section {
+  background-color: #fff;
+  margin: 0 23px;
+}
+
+.works-Section .recentWorks_header {
+  text-align: center;
+  padding-top: 120px;
+  padding-bottom: 104px;
+  color: #1c1a19;
+  font-size: 30px;
+  font-weight: 400;
+  letter-spacing: 1px;
+  font-family: 'Crete Round', serif;
+}
+
+.cards {
+  margin-top: 24px;
+}
+
+.works-Section .cards {
+  display: grid;
+}
+
+.card-title {
+  padding-top: 25px;
+  padding-bottom: 10px;
+  font-family: 'Crete Round', serif;
+  font-weight: 400;
+}
+
+.card1_description {
+  padding-bottom: 15px;
+  line-height: 23px;
+}
+
+.card1 .allButtons {
+  display: flex;
+  flex-direction: column;
+}
+
+.seeProject {
+  padding-top: 20px;
+}
+
+button {
+  cursor: pointer;
+  font-family: 'Roboto', sans-serif;
+}
```

Comment:
[OPTIONAL] As stated in the project requirements below, kindly implement all the button interactions (enable, hover, pressed). You can find them in the desktop side of the figma design.

![Screenshot from 2022-09-24 12-11-24](https://user-images.githubusercontent.com/65233860/192119226-6e69bbe7-f9f1-4725-9a1a-36aadc8e9a78.png)